core/api/gql/user_groups.go

```diff
@@ -101,21 +82,89 @@ func (m Mutation) CreateStaticUserGroup(
 		}
 	},
 ) StaticUserGroup {
-	panic("not implemented")
+	appID, internalErr := fromGraphQLID(args.AppID)
+	if internalErr != nil {
+		internalErr := errs.NewError(
+			errs.InvalidArgument,
+			internalErr.Error(),
+		)
+		m.deps.logger.ErrorWithContext(ctx, internalErr)
+		return StaticUserGroup{}
+	}
+
+	userIDs := make([]uint64, len(args.Input.UserIDs))
+	for _, userID := range args.Input.UserIDs {
+		id, err := fromGraphQLID(userID)
+		if err != nil {
+			internalErr := errs.NewError(
+				errs.InvalidArgument,
+				err.Error(),
+			)
+			m.deps.logger.ErrorWithContext(ctx, internalErr)
+			return StaticUserGroup{}
+		}
+		userIDs = append(userIDs, id)
+	}
+
+	createUserStaticGroupInput := service.CreateUserStaticGroupInput{
+		Name:    args.Input.Name,
+		UserIDs: userIDs,
+	}
+	group, err := m.deps.groupService.CreateStaticUserGroup(ctx, appID, createUserStaticGroupInput)
+	if err != nil {
+		m.deps.logger.ErrorWithContext(ctx, err)
+		return StaticUserGroup{}
+	}
+
+	return newStaticUserGroup(m.deps, group)
 }
 
 func (m Mutation) UpdateStaticUserGroup(
 	ctx context.Context,
 	args struct {
-		AppID   graphql.ID
 		GroupID graphql.ID
 		Input   struct {
 			Name    string
 			UserIDs []graphql.ID
 		}
 	},
-) StaticUserGroup {
-	panic("not implemented")
+) (StaticUserGroup, error) {
+	groupID, internalErr := fromGraphQLID(args.GroupID)
+	if internalErr != nil {
+		internalErr := errs.NewError(
+			errs.InvalidArgument,
+			internalErr.Error(),
+		)
+		m.deps.logger.ErrorWithContext(ctx, internalErr)
+		return StaticUserGroup{}, errs.ToResolverErr(internalErr)
+	}
+
+	userIDs := make([]uint64, len(args.Input.UserIDs))
+	for _, userID := range args.Input.UserIDs {
+		id, err := fromGraphQLID(userID)
+		if err != nil {
+			internalErr := errs.NewError(
+				errs.InvalidArgument,
+				err.Error(),
+			)
+			m.deps.logger.ErrorWithContext(ctx, internalErr)
+			return StaticUserGroup{}, errs.ToResolverErr(internalErr)
+		}
+		userIDs = append(userIDs, id)
+	}
+
+	updateUserStaticInput := service.UpdateUserStaticGroupInput{
```

Comment:
```suggestion
	updateStaticUserInput := service.UpdateStaticUserGroupInput{
```

Can we make the names consistent between API and inputs? Thanks!